app/src/main/java/org/oppia/android/app/databinding/ViewBindingAdapters.java

```diff
@@ -12,7 +12,9 @@
 public final class ViewBindingAdapters {
 
   /**
-   * BindingAdapter to set the height of a View.
+   * BindingAdapter to set the height of a View. If this value is calculated in data fetching, the
+   * layout will require a default value since data fetching is not immediately evaluated with
+   * view layouts and measurements.
```

Comment:
```suggestion
   * BindingAdapter to set the height of a View. If this value is calculated in data fetching, the
   * layout will require a default value since binding adapters aren't called until after initial
   * view measuring and layouting.
```

Slight wording adjustment to be a bit clearer/more correct on terminology.

Please update the similar method KDoc below.

Reply:
Done -- adjusted slightly since layouting is not a word :)

Reply:
It kind of is in Android (at least I use it as one). :)